space/authz/authz_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
 	"time"
 
+	netcontext "context"
```

Comment:
There's already an import of `context` in this file, so we should rather get rid of the `netcontext` alias, shouldn't we ?